_posts/en/newsletters/2023-04-12-newsletter.md

```diff
@@ -0,0 +1,181 @@
+---
+title: 'Bitcoin Optech Newsletter #246'
+permalink: /en/newsletters/2023/04/12/
+name: 2023-04-12-newsletter
+slug: 2023-04-12-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Splicing specification discussions:** several LN developers posted
+  to the Lightning-Dev mailing list this week about ongoing work on the
+  [draft specification][bolts #863] for [splicing][topic splicing],
+  which allows some funds in an offchain LN channel to be spent onchain
+  (splice-out) or for onchain funds to be added to an offchain channel
+  (splice-in).  The channel can continue to operate while an onchain
+  splicing transaction is waiting for sufficient confirmations.
+
+  {:.center}
+  ![Splicing transaction flow](/img/posts/2023-04-splicing1.dot.png)
```

Comment:
In the graphs, I interpret the boxes as transactions and the arrows as TXOs. I assume that the silver boxes are meant to indicate that transactions are confirmed. If all of these assumptions are correct, it might be useful to give the “New Funding” transaction another shading to distinguish that the “New Funding” tx is published to the blockchain immediately, while the commitment transactions are not.

Further, it seems to me that the important information is that there is two alternative paths for the funding transaction to be spent while the New Funding tx is waiting for confirmation. I would perhaps add the original path as an alternative spending path to the funding in the Splice graphs, e.g. for splice in (other parts on right not drawn):

```
┌───────────┐       ┌─────────────┐
│           ├──────►│             │
│ Funding   │       │ Comm (Orig) │
│           ├───┐   │             │
└───────────┘   │   └─────────────┘
                │
┌───────────┐   │   ┌─────────────┐
│           │   │   │             │
│   Alice   ├───┴──►│  New Funding│
│           │       │             │
└───────────┘       └─────────────┘
```